pkg/imgpkg/cmd/registry_flags.go

```diff
@@ -94,6 +73,25 @@ func (r *RegistryFlags) AsRegistryOpts() registry.Opts {
 	if os.Getenv("IMGPKG_ANON") == "true" {
 		opts.Anon = true
 	}
+	iaasAuth, found := os.LookupEnv("IMGPKG_ENABLE_IAAS_AUTH")
+	if found {
+		if iaasAuth == "true" {
```

Comment:
do we wanna remove the casing before the check? Like strings.ToLower() or something. So that even if I put `True` it works? Or is that going against the current way we check the env_vars?

Reply:
We were using != "false" so I think you are right, doing ToLower makes sense. Fixed